core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -458,7 +458,7 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal
 
     def deserializeEvent(row: Row): Any = {
       val event = serialization.deserialize(
-        row.getBytes("event").array,
+        Bytes.getArray(row.getBytes("event")),
```

Comment:
was this the only place, nothing in snapshots?

Reply:
Doh i am blind, i did grep for this too